jupyterhub/templates/image-puller/serviceaccount.yaml

```diff
@@ -2,18 +2,16 @@
 ServiceAccount for the pre-puller hook's image-awaiter-job
 */}}
 {{- if .Values.prePuller.hook.serviceAccount.create -}}
-{{- if (include "jupyterhub.imagePuller.daemonset.hook.install" .) -}}
+{{- if or (include "jupyterhub.imagePuller.daemonset.hook.install" .) .Values.prePuller.continuous.enabled -}}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: {{ include "jupyterhub.hook-image-awaiter-serviceaccount.fullname" . }}
   labels:
     {{- include "jupyterhub.labels" . | nindent 4 }}
-    hub.jupyter.org/deletable: "true"
```

Comment:
We don't want to delete the service account if the daemonsets are enabled

Reply:
Does this need to be conditional on whether daemonset-hook but not daemonset-continuous is enabled?

Reply:
I don't think so. 

We don't want to delete the service account if either daemonset-hook or daemonset-continuous is enabled.